lib/model/store.dart

```diff
@@ -345,6 +345,64 @@ abstract class GlobalStore extends ChangeNotifier {
     ));
   }
 
+  /// Attempt to refresh the realm metadata for each account.
+  ///
+  /// Fetches server settings for each account and then calls [updateRealmData]
+  /// to update the realm metadata (name and icon).
+  ///
+  /// The updation for some accounts may be skipped if:
+  ///  - The [PerAccountStore] for the account is already loaded or is loading.
+  ///  - The version of corresponding realm server is unsupported.
+  ///
+  /// Returns immediately; the fetches and updates are done asynchronously.
+  void refreshRealmMetadata() {
+    for (final accountId in accountIds) {
+      // Avoid updating realm metadata if per account store has already
+      // loaded or is being loaded. It should be updated with data from
+      // the initial snapshot or realm update events.
+      if (_perAccountStoresLoading.containsKey(accountId)) continue;
+      if (_perAccountStores.containsKey(accountId)) continue;
```

Comment:
Here "should" sounds to me like it's saying there's some obligation which has yet to be discharged. I think what this is intended to say is a reassurance: the update will get taken care of, by other code.
```suggestion
      // Avoid updating realm metadata if per account store has already
      // loaded or is being loaded. It will be updated with data from
      // the initial snapshot or realm update events.
      if (_perAccountStoresLoading.containsKey(accountId)) continue;
      if (_perAccountStores.containsKey(accountId)) continue;
```